### PR TITLE
winlogbeat-os - (few changes)

### PR DESCRIPTION
**Pull request by Avni3**
Added new field OriginalFileName for event_id=1
Added new fields Protocol and User for event_id=3
Added new fields ImageLoaded, SignatureStatus, Signature and Signed for event_id=6
Added new fields Image, ImageLoaded for event_id=7
Added new field Operation for event_id=20

Samiksha -
1.I have added few sysmon event id. 
2.checked stream and annotation of that particular stream
3.added annotation which are recently added into blueprint.
4. changes action name from NET_SHR_ACS_CHK to NET_SHARED_ACCESS_CHECK
5. Removed some annotation which are not part of blueprint.

6. added few event id in the section of  first match \"Microsoft\-Windows\-Security\-Auditing\"
added stream - WIN-AUDIT, THREAT, CONFIG ,

### CODE REVIEW

File: extractors/OS/ms-windows-winlogbeat.yaml [hunk 1]
```diff
@@ -316,13 +492,17 @@ event-details:
         Status: PASSED
```
Comment (Sherin18) [inline extractors/OS/ms-windows-winlogbeat.yaml:492]: Please check the all event id mapping .4662 event is generated when a user accesses an AD object.

Comment (samiksha2005) [reply extractors/OS/ms-windows-winlogbeat.yaml:492]: yes i will change the action using ref docs, and adding few events which are present in nxlog extractor.

File: extractors/OS/ms-windows-winlogbeat.yaml [hunk 2]
```diff
@@ -294,6 +471,8 @@ event-details:
...
```
Comment (Sherin18) [inline extractors/OS/ms-windows-winlogbeat.yaml:471]: AUDIT_LOG_CLEARED should fall under Win Audit stream

Comment (samiksha2005) [reply extractors/OS/ms-windows-winlogbeat.yaml:471]: yes , i have changed it to stream- win-audit with action audit_log_cleared and category-policy.

**Comment by Sherin18** [comment]
Validated SYSMON-PROCESS,SYSMON-NETWORK,SYSMON-FILE,SYSMON-REGISTRY,SYSMON-PIPE,IAM,SYSMON-IMAGE-LOAD,SYSMON-DRIVER-LOAD,SYSMON-DNS,Authentication
ON Hold SYSMON-WMI,SYSMON-SERVICE,WIN-AUDIT,CONFIGURATION,THREAT

**Pull request merged**